docs/book/installation.rst

```diff
@@ -87,6 +38,12 @@ place, run the following commands:
 
 This package has the whole ``sylius/sylius`` package in vendors, so you can easily update it and focus on your custom development.
 
+.. warning::
+
+    During the ``sylius:install`` command you will be asked to provide important information, but also its execution ensures
+    that the default **currency** (USD) and the default **locale** (English - US) are set.
```

Comment:
OK, the next sentence clarifies it a bit, so let's just add a note that these defaults are configurable in ``parameters.yml``.